C_CPP/Exception Handling/Exception_Handling_in_C.md

```diff
@@ -0,0 +1,131 @@
+# Exception Handling in C
+Exception handling in C is very difficult as C does not provide any direct support to handle the exception handling , there are some ways by which programmer can handle the exceptions .
```

Comment:
don't use 'very difficult', use 'comparatively tough'

Reply:
> don't use 'very difficult', use 'comparatively tough'

Changes applied, please review it again @apeksha235.